deploy/helm/sumologic/conf/metrics/otelcol/config.yaml

```diff
@@ -1 +1,346 @@
+      - value: /prometheus.metrics.kubelet
+        exporters:
+          - sumologic/kubelet
+      ## node metrics
+      - value: /prometheus.metrics.node
+        exporters:
+          - sumologic/node
+      ## scheduler metrics
+      - value: /prometheus.metrics.scheduler
+        exporters:
+          - sumologic/scheduler
+      ## state metrics
+      - value: /prometheus.metrics.state
+        exporters:
+          - sumologic/state
+
+  ## Configuration for Memory Limiter Processor
+  ## The memory_limiter processor is used to prevent out of memory situations on the collector.
+  ## ref: https://github.com/SumoLogic/opentelemetry-collector/tree/main/processor/memorylimiter
+  memory_limiter:
+    ## check_interval is the time between measurements of memory usage for the
+    ## purposes of avoiding going over the limits. Defaults to zero, so no
+    ## checks will be performed. Values below 1 second are not recommended since
+    ## it can result in unnecessary CPU consumption.
+    check_interval: 5s
+
+    ## Maximum amount of memory, in %, targeted to be allocated by the process heap.
+    limit_percentage: 75
+    ## Spike limit (calculated from available memory). Must be less than limit_percentage.
+    spike_limit_percentage: 20
+
+  sumologic_schema:
+    add_cloud_namespace: false
+
+  ## Configuration for Batch Processor
+  ## The batch processor accepts spans and places them into batches grouped by node and resource
+  ## ref: https://github.com/SumoLogic/opentelemetry-collector/blob/main/processor/batchprocessor
+  batch:
+    ## Number of spans after which a batch will be sent regardless of time
+    send_batch_size: 1_024
+    ## Time duration after which a batch will be sent regardless of size
+    timeout: 1s
+  ## Configuration for Kubernetes Processor
+  ## ref: https://github.com/SumoLogic/sumologic-otel-collector/tree/main/pkg/processor/k8sprocessor
+  k8s_tagger:
+    ## Has to be false to enrich metadata
+    passthrough: false
+    owner_lookup_enabled: true  # To enable fetching additional metadata using `owner` relationship
+    extract:
+      metadata:
+        ## extract the following well-known metadata fields
+        - daemonSetName
+        - deploymentName
+        - nodeName
+        - replicaSetName
+        - serviceName
+        - statefulSetName
+      labels:
+        - tag_name: "pod_labels_%s"
+          key: "*"
+      delimiter: "_"
+    pod_association:
+      - from: build_hostname  # Pods are identified by Pod name and namespace
+  ## Configuration for Source Processor
+  ## Source processor adds Sumo Logic related metadata
+  ## ref: https://github.com/SumoLogic/sumologic-otel-collector/tree/main/pkg/processor/sourceprocessor
+  source:
+    collector: {{ .Values.sumologic.collectorName | default .Values.sumologic.clusterName | quote }}
+service:
+  telemetry:
+    logs:
+      level: {{ .Values.metadata.metrics.logLevel }}
+  extensions:
+    - health_check
+    - file_storage
```

Comment:
What about `{{ if .Values.metadata.persistence.enabled }}` that's used in the [logs PR](https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/2626/files#diff-14625ea0fb1afc1752d344678aa17df4e9618ecf05f944046c44ded34326d474R339)?

Reply:
Good point, I'll add it. I submitted this PR first and took more time to think about the conditionals in #2626